x-pack/plugins/ml/public/application/data_frame_analytics/pages/analytics_creation/components/advanced_step/advanced_step_form.tsx

```diff
@@ -47,6 +132,11 @@ export const AdvancedStepForm: FC<CreateAnalyticsStepProps> = ({
   const [advancedParamErrors, setAdvancedParamErrors] = useState<AdvancedParamErrors>({});
   const [fetchingAdvancedParamErrors, setFetchingAdvancedParamErrors] = useState<boolean>(false);
 
+  const {
+    services: { docLinks },
+  } = useMlKibana();
+  const { ELASTIC_WEBSITE_URL, DOC_LINK_VERSION } = docLinks;
```

Comment:
we pull doc links out like this in a few places.
It might be worth adding a specialised `useDocLinks` hook.
(not suggesting it for this PR)